### PR TITLE
Feature/wbtech 197 integrate all calculator required for calculation

### PR DESCRIPTION
**Pull request by priyabyanjankar**
Changes made are 
 calculated how much deposit  required using stamp duty, fhog, lmi existing api  and lvr 

To run test Cases : 
yarn test

### CODE REVIEW

File: .serverless/cloudformation-template-create-stack.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
```
Comment (ravi-hle) [inline .serverless/cloudformation-template-create-stack.json:2]: Keep .serverless files in the gitignore.

File: .serverless/cloudformation-template-create-stack.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{
```
Comment (amirsaga) [inline .serverless/cloudformation-template-create-stack.json:1]: .serverless directory should be in .gitignore.

File: .env [hunk 3]
```diff
@@ -0,0 +1 @@
+REACT_APP_ENV=DEV
```
Comment (amirsaga) [inline .env:1]: @priyabyanjankar create a .env-example file instead of commiting .env file

**Review by ravi-hle — COMMENTED** [review]
LGTM

File: .env (hunk 3 above)
Comment (priyabyanjankar) [reply .env:1]: @amirsaga dai mentioned changes are done creating example file and serverless in gitignore

File: src/components/Deposit/DepositResult/DepositFundCost.js [hunk 4]
```diff
@@ -59,85 +47,53 @@ const EstimatedCost = ({ estimateCost }) => {
...
-                                    <NumberFormat
-                                        value='350'
-                                        thousandSeparator={true}
-                                        decimalScale={2}
-                                        fixedDecimalScale={true}
-                                        displayType={'text'}
-                                        prefix={'$'}
-                                    />
-                                </td>
+                                <td className='w-30'> {`$${amount.LENDER_FEE}.00`}</td>
```
Comment (amirsaga) [inline src/components/Deposit/DepositResult/DepositFundCost.js:92]: `$${amount.LENDER_FEE}.00` can be better kept as (amount.LENDER_FEE).toFixed(2)

File: src/components/Deposit/DepositResult/DepositFundCost.js [hunk 5]
```diff
@@ -59,85 +47,53 @@ const EstimatedCost = ({ estimateCost }) => {
...
-                                    <NumberFormat
-                                        value='2000'
-                                        thousandSeparator={true}
-                                        decimalScale={2}
-                                        fixedDecimalScale={true}
-                                        displayType={'text'}
-                                        prefix={'$'}
-                                    />
-                                </td>
+                                <td className='w-30'> {`$${amount.LEGAL_COST}.00`}</td>
```
Comment (amirsaga) [inline src/components/Deposit/DepositResult/DepositFundCost.js:96]: Similar (amount.LEGAL_COST).toFixed(2)

File: .env (hunk 3 above)
Comment (amirsaga) [reply .env:1]: you need to delete and commit the files again in order to not track the files later mentioned on gitignore.

File: src/components/Deposit/DepositResult/DepositFundCost.js (hunk 5 above)
Comment (priyabyanjankar) [reply src/components/Deposit/DepositResult/DepositFundCost.js]: @amirsaga  dai  above mentioned are done.

**Review by amirsaga — APPROVED** [review]
LGTM

**Pull request merged**